src/Groups/action.jl

```diff
@@ -1,13 +1,13 @@
-"""
-A *group action* of a group G on a set Ω (from the right) is defined by
-a map μ: Ω × G → Ω that satisfies the compatibility conditions
-μ(μ(x, g), h) = μ(x, g*h) and μ(x, one(G)) == x for all x ∈ Ω.
+raw"""
```

Comment:
@ThomasBreuer this free standing string duplicates parts of `action.md` but is not attached to any object, so as far as I know, it is not reachable from the REPL. So what is the point of having it around? Am I missing something?

Reply:
This definition was the starting point of the file. As far as I remember, we did not have `action.md` at that time.
We can remove the string.